Makefile

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env make
 
-.PHONY: prep build releases compile images kube kube-dist helm publish run stop dist generate mysql-release uaa-release scf-helper-release
+.PHONY: prep build compile images kube kube-dist helm publish run stop dist generate mysql-release uaa-release
```

Comment:
Why are `mysql-release` and `uaa-release` still there? They are not being used afaict and should be removed as well.

Reply:
No idea, probably left overs. Removing.